utils/bigclown/bigclown-mqtt2influxdb/Makefile

```diff
@@ -0,0 +1,48 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bigclown-mqtt2influxdb
+PKG_VERSION:=1.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/bigclownlabs/bch-mqtt2influxdb/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=5be14132311e85215abbfd732fe6cd652522ea0a343ee8ba7abab3ec7578eb99
+PKG_LICENSE:=MIT
+PKG_BUILD_DIR:=$(BUILD_DIR)/bch-mqtt2influxdb-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../../lang/python/python3-package.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=BigClown
+  URL:=https://github.com/bigclownlabs/bch-mqtt2influxdb
+  TITLE:=BigCLown MQTT to Influxdb bridge
+  DEPENDS:=+python3-paho-mqtt +python3-yaml +python3-influxdb +python3-jsonpath-ng +python3-schema
+endef
+
+define Py3Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/init $(1)/etc/init.d/bigclown-mqtt2influxdb
+	$(INSTALL_DATA) ./files/config.yml $(1)/etc/bigclown-mqtt2influxdb.yml
+endef
+
+define Package/$(PKG_NAME)/conffiles
+/etc/bc-mqtt2influxdb.yml
+endef
+
+define Build/Compile
+	sed -i 's/@@VERSION@@/$(PKG_VERSION)/' "$(PKG_BUILD_DIR)/setup.py"
```

Comment:
this looks hacky;
is there another way to do it without `sed` ?
using patches, also is a bit weird, since this does a parametrized replace/sed ;

Reply:
I can use awk if you want. :smile: 

Well unfortunately bigclown upstream have this in any release and they replaces package version in that. I think that it is pretty stupid but I have to somehow put version there and only way is to edit it. You correctly pointed out patches but version is known only in package it self. Having version also in patch is not intuitive and I am sure that someone in future would forget to bump it as well. This does make sense to me, although I accept that it is nasty way. I don't know better way non the less.

Reply:
I took a look as well.
Yep. Let's leave it like this.

It's crap, but it's bigclown's fauls